xlr/sdk/src/types.ts

```diff
@@ -1,6 +1,7 @@
 import type { Node } from "jsonc-parser";
+import { DiagnosticSeverity } from "vscode-languageserver-types";
```

Comment:
I'm not exactly sure how we would solve but but if feels kinda gross to pull in this whole package just for these enums.